src/HuC3.md

```diff
@@ -0,0 +1,171 @@
+  $B  | RTC command/argument (write)
+  $C  | RTC command/response (read)
+  $D  | RTC semaphore (read/write)
+  $E  | IR (read/write)
+
+If any other value is written, reads from A000–BFFF return open bus
+values (often $FF, but not guaranteed), and writes are ignored.
+
+### 2000–3FFF – ROM Bank Number (Write-Only)
+
+HuC-3 can accept a 7-bit bank number here.
+
+### 4000–5FFF – RAM Bank Select (Write-Only)
+
+HuC-3 can accept a bank number of at least 2 bits here.
+
+### 6000–7FFF – Nothing? (Write-Only)
+
+Games write $01 here on startup, but it doesn\'t seem to have any
+observable effect.
+
+
+## I/O Registers
+
+These registers are accessed by reading or writing int the A000–BFFF
+range after setting the RAM/RTC/IR Select register. Address lines
+A12–A0 are not connected to the HuC-3 chip, so the offset within the
+range is ignored. Data line D7 is not connected to the HuC-3 chip, so
+the most significant bit of reads always returns an open bus value
+(usually high), and the most significant bit of writes is always
+ignored.
+
+The RTC MCU communication protocol is described below.
+
+### B – RTC Command/Argument (Write)
+
+The value written consists of a command in bits 6-4, and an argument
+value in bits 4–0. For example the value $62 is command $6 with argument
+value $2. Writing to this register just sets the values in the mailbox
+registers – it does not cause the command to be executed.
+
+### C – RTC Command/Response (Read)
+
+When read, bits 6–4 return the last command written to register $B, and
+bits 3–0 contain the result from the last command executed.
+
+### D – RTC Semaphore (Read/Write)
+
+When reading, the least significant bit is high when the RTC MCU is
+ready to receive a command, or low when the RTC MCU is busy.
+
+Writing with the least significant bit clear requests that the RTC MCU
+execute the last command written to register $B.
+
+### E – IR (Read/Write)
+
+Similar to the equivalent register of the HuC-1. The least significant
+bit is used for infrared communication.
+
+
+## RTC Communication Protocol
+
+The HuC-3 chip provides read and write access to a 256-nybble window
+into the RTC MCU\'s internal memory. Multi-nybble values are stored
+with the least significant nybble at the lowest address (little Endian).
+There are commands for setting the access address, reading and writing
+values, and a few higher-level operations.
+
+Games use the following sequence to execute a command:
+
+* Write $0D to 0000 (select RTC Semaphore register)
+* Poll A000 until least significant bit is set (wait until ready)
+* Write $0B to 0000 (select RTC Command/Argument register)
+* Write command and argument to A000
+* Write $0D to 0000 (select RTC semaphore register)
+* Write $FE to A000 (clear semaphore, requesting MCU execute command)
+* Poll A000 until least significant bit is set (wait for completion)
+* Write $0C to 0000 (select RTC Command/Response register)
+* Read A000 and use value from 4 least significant bits
```

Comment:
```suggestion
* Write $0D to $0000 (select RTC Semaphore register)
* Poll $A000 until least significant bit is set (wait until ready)
* Write $0B to $0000 (select RTC Command/Argument register)
* Write command and argument to $A000
* Write $0D to $0000 (select RTC semaphore register)
* Write $FE to $A000 (clear semaphore, requesting MCU execute command)
* Poll $A000 until least significant bit is set (wait for completion)
* Write $0C to $0000 (select RTC Command/Response register)
* Read $A000 and use value from 4 least significant bits
```

Reply:
Once again, I was following the rule I quoted from CONTRIBUTING.md that says to omit the radix prefix for addresses.